src/admin.rs

```diff
@@ -161,10 +162,20 @@ fn dump_cert(x509: &X509) -> CertDump {
 
 async fn dump_certs(cert_manager: &SecretManager) -> Vec<CertsDump> {
     let mut dump = cert_manager
-        .collect_certs(|id, certs| CertsDump {
-            identity: id.to_string(),
-            ca_cert: [dump_cert(certs.x509())],
-            cert_chain: certs.iter_chain().map(dump_cert).collect(),
+        .collect_certs(|id, certs, state| {
+            let mut dump = CertsDump {
+                identity: id.to_string(),
+                state,
+                ca_cert: [CertDump {
```

Comment:
nit: do we just want empty vec if there is none?

Reply:
cc @qfel 

The `ca_cert` field is an array (with set size) and not a Vec, I know Piotr did this to be in line with Envoy API